src/shared/components/Menu/Menu.tsx

```diff
@@ -68,36 +68,64 @@ export const Menu = <T = unknown,>({
             action?.({ title, context });
             onClose();
           };
+
+          const TooltipWrapper = ({ children }: { children: JSX.Element }) =>
+            tooltip ? <Tooltip tooltipTitle={tooltip}>{children}</Tooltip> : children;
+
           const menuItemContent = (
-            <StyledMenuItemContent customItemColor={customItemColor}>
-              {icon && <ListItemIcon>{icon}</ListItemIcon>}
-              {!!title && (
-                <StyledBodyLarge
-                  color={customItemColor || variables.palette.on_surface}
-                  letterSpacing="xxl"
-                >
-                  {t(title)}
-                </StyledBodyLarge>
-              )}
-            </StyledMenuItemContent>
+            <TooltipWrapper>
+              <StyledMenuItemContent customItemColor={customItemColor}>
+                {icon && <ListItemIcon>{icon}</ListItemIcon>}
+                {!!title && (
+                  <StyledBodyLarge
+                    color={customItemColor || variables.palette.on_surface}
+                    letterSpacing="xxl"
+                    whiteSpace="pre-line"
+                  >
+                    {t(title)}
+                  </StyledBodyLarge>
+                )}
+              </StyledMenuItemContent>
+            </TooltipWrapper>
           );
 
-          return type === MenuItemType.Divider ? (
-            <Divider key={index} sx={{ flex: 1, my: 0.8 }} data-testid={dataTestId} />
-          ) : (
-            <MenuItem
-              key={index}
-              disabled={disabled}
-              onClick={disabled ? undefined : handleMenuItemClick}
-              data-testid={dataTestId}
-            >
-              {tooltip ? (
-                <Tooltip tooltipTitle={tooltip}>{menuItemContent}</Tooltip>
-              ) : (
-                menuItemContent
-              )}
-            </MenuItem>
-          );
+          switch (type) {
+            case MenuItemType.Normal:
+            default:
+              return (
+                <MenuItem
+                  key={index}
+                  data-testid={dataTestId}
+                  disabled={disabled}
+                  onClick={disabled ? undefined : handleMenuItemClick}
+                >
+                  {menuItemContent}
+                </MenuItem>
+              );
+            case MenuItemType.Info:
+              return (
+                <MenuItem
+                  key={index}
+                  data-testid={dataTestId}
+                  onClick={undefined}
+                  disableTouchRipple
+                  disableRipple
+                  sx={{
+                    '&&': {
+                      cursor: 'default',
+                      userSelect: 'text',
+                      ':hover': {
+                        backgroundColor: 'inherit',
+                      },
+                    },
+                  }}
```

Comment:
Unfortunately, this isn't totally sufficient for making this a non-selectable/"read-only" element.

`MenuItem` will render a list item with `tabIndex` and `role` attributes that makes this focusable and suggests interaction, and the hover styles we're overriding here will apply when the element is focused (eg, by clicking or via keyboard navigation).

Here's an example of how the macOS screen reader will report this element, for example:
 
![Screenshot 2024-05-31 at 11 25 46 AM](https://github.com/ChildMindInstitute/mindlogger-admin/assets/1670836/75018bcc-d872-4433-b41f-9ed4d0b31bab)

I tried to figure out a solution locally by passing a `component` prop that just strips out the relevant props that MenuItem would attempt to render, but doing that seems to mess up the keyboard support and focus management this component has going on.

Ultimately, I think we should just not use the `MenuItem` component at all here, and instead handle this whole case by doing something like:

```jsx
case MenuItemType.Info:
  return (
    <Box data-testid={dataTestId} sx={{ px: 2, py: 1.2 }}>
      {menuItemContent}
    </Box>
  );
```

This is actually what the MUI-Core docs suggest for handling non-interactive content in a `Menu`, [in this example](https://mui.com/base-ui/react-menu/#wrapping-menu-items)

Reply:
That's a good suggestion, thanks. I started by conditionally passing props to the `MenuItem` before I ultimately created a separate `MenuItemType`, so it makes sense to use a different component here.